elastio-stream-kafka/README.md

```diff
@@ -57,4 +59,69 @@
+        4. In the "Attach permissions policies" section, search for ```AmazonECSTaskExecutionRolePolicy, SecretsManagerReadWrite, AmazonS3FullAccess, ECR_FullAccess, ElastioFullAdmin``` select "Policies", then choose "Next": "Tags".
+        5. For Add tags (optional), specify custom tags to associate with the policy and then choose "Next": "Review".
+        6. For Role name, type `ecs_task_execution_role` and choose "Create role".
+
+3. Build and push a docker image:
+    > Before you start this section, review the dockerfile and read its comments.
+    > You can find the latest version by following this link. https://gallery.ecr.aws/elastio-dev/elastio-cli
+    1. Edit file contrib/elastio-stream-kafka/docker_backup_config.py and put data about topic, brokers, vault.
+    2. Create ECR respository.
+    3. Register your docker cli by getting the ecr public image with the command:
+        ```aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws```
+        
+    4. Open the ECR repository you have created earlier and press the button "View push commands". The screen will list four commands. Run them in your terminal to build the docker image.
+4. Create a Fargate Cluster.
+    1. From the AWS console, select `Elastic Container Service`=>`Create Cluster`.
+    2. Under "Select" cluster template, select "networking only".
+    3. Enter your cluster name and press `Create`.
+5. Create an ECS Task.
+    1. Select `Task Definitions` from the left-side menu. Then choose `Create new task definition`.
+    2. Select Fargate => "Next Step".
+    3. Enter `Task definition name`.
+    4. In the `Task role` field choose `ecs_task_execution_role`.
+    5. Choose `Linux` as an operating system family.
+    6. Task execution role.
+    7. Task memory (GB): 4GB
+    8. Task CPU (vCPU): 1vCPU
+    9. Select `Add container`.
+    10. Enter a container name.
+    11. In the image field put URL of your docker image from ECR repository. Copy the arn from the ECS tab.
+    12. Set `Private repository authentication` to true
+    13. Memory Limits (MiB):
+        1. Soft limit: `4096`
+        2. Hard limit: `4096`
+    14. Press `Add`.
+    15. Lastly, press `Create` button.
+6. Run the ECS schedule task.
+    1. Select your cluster in ECS Clusters.
+    2. In the navigation tab choose `Scheduled Tasks`.
+    3. Press `Create` button.
+    4. Enter `Schedule rule name`=>`Target id`.
+    5. Launch type choose `Fargate`.
+    6. Task Definition, choose your task definition.
+    > If you want to backup a topic from the AWS MSK, select the same VPC as you selected for MSK. Task Security Group has to accept Kafka traffic.
+    7. Select a VPC from the list in the Cluster VPC field.
+    8. Add at least one subnet.
+    9. Enable Auto-assign public IP.
+    10. Press `Create` button. This will run the task according to the selected schedule.
```

Comment:
Will this push logs anywhere? I feel like this needs a follow-up section describing how to see the logs for troubleshooting.